### DISCUSSION TITLE
Deboot als mobile "Webcam"/Zugriff auf Video-Stream

### DISCUSSION THREAD

**Discussion by SmartNightly** [mrbungle64/ioBroker.ecovacs-deebot#312] [Ideas]
Würde gerne den Roboter als mobile "Webcam" benutzen. Bspw. anhand eines Triggers, wenn niemand zu Hause und Bewegungsmelder Bewegung erkennt, Robi an Position X platzieren, Licht an, Video Stream aufzeichnen (bspw. auf Synology Rackstation) und Telegram Meldung mit Bild senden.

Ist bekannt, wie der Zugriff auf den Videostream/Bilder möglich ist? (Deebot T9 Aivi) Habe bis jetzt diesbezüglich keine Infos gefunden.

Evtl. über Datenpunkte im Adapter Bilder und Video-URLs verfügbar machen.

**Comment by mrbungle64** [comment]
Hallo @SmartNightly 

> Würde gerne den Roboter als mobile "Webcam" benutzen. Bspw. anhand eines Triggers, wenn niemand zu Hause und Bewegungsmelder Bewegung erkennt, Robi an Position X platzieren, Licht an, Video Stream aufzeichnen (bspw. auf Synology Rackstation) und Telegram Meldung mit Bild senden.

ja, das klingt spanned und könnte auch früher oder später mal umgesetzt werden

> Ist bekannt, wie der Zugriff auf den Videostream/Bilder möglich ist? (Deebot T9 Aivi) Habe bis jetzt diesbezüglich keine Infos gefunden.

Bisher kenne ich auch keine Möglichkeit - meine Versuche mit mitmproxy haben leider bei meinem T8 AIVI dazu geführt, dass der Roboter danach nicht mehr erreichbar war. Entweder hat der sich dann jedes mal irgendwie "verschluckt" oder das ist eine Art Security-Feature, dass der sich bei "auffälligem" Zugriff auf die Kamera-Funktion für eine gewisse Zeit abschaltet. Habe das seit dem nicht mehr probiert.
